agendas/2019-07-03.md

```diff
@@ -39,6 +39,7 @@ Lee Byron            | GraphQL Foundation | Menlo Park, CA
 
 1. Opening of the meeting
 1. Introduction of attendees
+1. Review [previous meeting's action items](../notes/2019-06-06.md) and check progress on those
```

Comment:
Would be great to link directly to "Action item" section, e.g. https://github.com/graphql/graphql-wg/blob/master/notes/2019-06-06.md#action-items

Reply:
@IvanGoncharov great idea! I've updated all occurrences to include the anchor 👍